index.html

```diff
@@ -820,13 +820,25 @@ <h3>JsonWebKey</h3>
               </dd>
               <dt><dfn class="lint-ignore">publicKeyJwk</dfn></dt>
               <dd>
+                <p>
 The `publicKeyJwk` property is REQUIRED, and its value MUST
 be a JSON Web Key that conforms to [[RFC7517]].
 It is RECOMMENDED that verification methods that use
 JWKs [[RFC7517]] to represent their <a>public keys</a> use the value of `kid` as
 their fragment identifier. It is RECOMMENDED that JWK `kid` values be set to
 the public key fingerprint [[RFC7638]]. See the first key in the example below
 for an instance of a public key with a compound key identifier.
+                </p>
+                <p>
+The `alg` property identifies the algorithm intended
+for use with the public key. Although optional, it is RECOMMENDED that `alg`
+be included, to avoid security issues arising from using the same key with multiple
+algorithms. If describing a key using an elliptic curve, the `crv` property is used
+to identify the particular curve type of the public key. The `kid` property is a hint
```

Comment:
It's not clear whether `crv` is optional or required. I've guessed optional.
```suggestion
algorithms. If describing a key using an elliptic curve, the OPTIONAL `crv` property is used
to identify the particular curve type of the public key. The OPTIONAL `kid` property is a hint
```